packages/suite/src/components/firmware/FirmwareInitialStandalone.tsx

```diff
@@ -61,7 +61,7 @@ export const FirmwareInitialStandalone = ({
     }
 
     // Bitcoin-only firmware is only available on T2T1 from v2.0.8 - older devices must first upgrade to 2.1.1 which does not have a Bitcoin-only variant
-    const isBitcoinOnlyAvailable = !!device.firmwareRelease?.release.url_bitcoinonly;
+    const isBitcoinOnlyAvailable = !!device.firmwareReleaseConfig?.isBitcoinOnlyAvailable;
```

Comment:
well, so maybe `isBitcoinOnlyAvailable` is still needed - there was this concept of 'incremental update' for older model Ts. it was not possible to go directly to the latest one. but maybe this concept has been obsoleted by some intermediary fw? who knows

Reply:
Interesting, I will get deeper into this scenario. There is no intermediary FW for model Ts so this might be an issue.